ee/tables/firmwarepasswd/firmwarepasswd.go

```diff
@@ -90,42 +90,29 @@ func (t *Table) generate(ctx context.Context, queryContext table.QueryContext) (
 	return []map[string]string{result}, nil
 }
 
-func (t *Table) runFirmwarepasswd(ctx context.Context, subcommand string, output *bytes.Buffer) error {
-	ctx, cancel := context.WithTimeout(ctx, 1*time.Second)
-	defer cancel()
-
-	cmd, err := allowedcmd.Firmwarepasswd(ctx, subcommand)
-	if err != nil {
-		return fmt.Errorf("creating command: %w", err)
-	}
-
+func (t *Table) runFirmwarepasswd(ctx context.Context, subcommand string) ([]byte, error) {
```

Comment:
this is not quite the same as it was, should probably leave it as close as possible for a large refactor